src/metabase/driver/workarounds.clj

```diff
@@ -0,0 +1,21 @@
+(ns metabase.driver.workarounds
+  "Driver specific workarounds. These extensions points should not normally be
+  used, as the default implementations cover normal drivers."
+  (:require [metabase.db.metadata-queries :as metadata-queries]
+            [metabase.driver :as driver]))
+
+(defmulti table-rows-sample
+  "Processes a sample of rows produced by `driver`, from the `table`'s `fields`
+  using the query result processing function `rff`.
+  The default implementation runs a row sampling MBQL query using the regular
+  query processor to produce the sample rows.
+  `opts` is a map that may contain additional parameters:
+  `:truncation-size`: size to truncate text fields to if the driver supports
+  expressions."
+  {:arglists '([driver table fields rff opts]), :style/indent 1}
```

Comment:
```suggestion
  {:arglists '([driver table fields rff opts]), :added "0.46.0"}
```

- This is not a macro or special form that should be indented with one special arg, `:style/indent 1` here is inappropriate. I know a bunch of other stuff incorrectly has that, but let's not perpetuate the weirdness in new stuff. Instead, let's fix the wrong indentation specs where we see them
- For driver methods it's nice to add what version we added the method so driver authors who want to write drivers that work across multiple Metabase versions can have an easier time doing so